mmv1/third_party/terraform/services/vmwareengine/resource_vmwareengine_private_cloud_test.go

```diff
@@ -20,20 +17,20 @@ func TestAccVmwareenginePrivateCloud_vmwareEnginePrivateCloudUpdate(t *testing.T
 	t.Parallel()
 
 	context := map[string]interface{}{
-		"region":          "southamerica-east1",
-		"random_suffix":   acctest.RandString(t, 10),
+		"region":        "southamerica-west1",
```

Comment:
Nit: I assume this is intentional, but I'm curious why it needed to change.

Reply:
southamerica-west1 is a region of low node utilisation, and changed it when I was facing some availability errors. It shouldn't make a difference though.

Reply:
Ok, makes sense.